routes/web.php

```diff
@@ -13,6 +13,6 @@
 |
 */
 
-Route::get('/', function () {
-    return view('welcome');
-});
+Route::resource('/tasks', 'TaskController')->only([
```

Comment:
`resource` có tác dụng gì?

Reply:
resource trong route đăng ký route cho resource controller ( controller có thể xử lý toàn bộ http request ).